src/Files.Core.SourceGenerator/Extensions/StringBuilderExtensions.cs

```diff
@@ -31,7 +31,7 @@ internal static StringBuilder AppendAutoGeneratedHeader(this StringBuilder sb, s
 		internal static StringBuilder AppendLicenceHeader(this StringBuilder sb)
 		{
 			return sb.AppendLine($"// Copyright (c) {DateTime.Now.Year} Files Community")
```

Comment:
```suggestion
			return sb.AppendLine($"// Copyright (c) 2018-2024 Files Community")
```

Reply:
```suggestion
			return sb.AppendLine($"// Copyright (c) 2018-{DateTime.Now.Year} Files Community")
```
This would be more appropriate. I'll add that now.